format_values.md

```diff
@@ -94,4 +92,4 @@ The following characters cannot be present in any fields, unless exempted.
 - U+000A (LINE FEED)
 - U+0085 (NEXT LINE)
 
-**TODO: I am bad at this. Need a better list.**
\ No newline at end of file
+**TODO: I am bad at this. Need a better list.**
```

Comment:
Why has this line changed?

Reply:
Good question. Probably a new-line character. I’ll fix that.